### DISCUSSION TITLE
Where is the right place to find official loxilb docker image ?

### DISCUSSION THREAD

**Discussion by TrekkieCoder** [loxilb-io/loxilb#53] [General]
I could find package here https://github.com/orgs/loxilb-io/packages as well as in https://hub.docker.com/repository/docker/loxilbio/loxilb.

Which one should I use ??

**Comment by ghost** [comment]
Currently, the packages in github are the docker images built from latest source (triggered by any push or pull request).  Docker Hub images are supposed to be stable release images. In the near future, we will host all images in github.
